release.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -o errexit -o pipefail
-
+an
```

Comment:
Anagram Generator ? Really ?
Not sure it is installed by default in every linux distibution...

Reply:
fixed